build_libzmq.sh

```diff
@@ -17,7 +17,7 @@ export CFLAGS=-fPIC
 export CXXFLAGS=-fPIC
 export PKG_CONFIG_PATH=$ZMQ_PREFIX/lib/pkgconfig
 
-test -f zeromq-$ZMQ.tar.gz || wget https://github.com/$ZMQ_REPO/releases/download/v$ZMQ/zeromq-$ZMQ.tar.gz -O zeromq-$ZMQ.tar.gz
+test -f zeromq-$ZMQ.tar.gz || ZMQ=$ZMQ ZMQ_REPO=$ZMQ_REPO node "$(dirname $(realpath $0))/download-zmq.js"
```

Comment:
@lgeiger This is the line that's causing Travis to fail. It's most likely the `"$(dirname $(realpath $0))/download-zmq.js"`. I tried `./download-zmq.js` and `PWD/download-zmq.js` but this is the only combo that worked for me locally.